src/Screens/ConfirmationScreen/confirmationPage.tsx

```diff
@@ -1,10 +1,233 @@
+      const allIngredients = cartElement.dish.ingredients.split(";");
+      const excludedIngredients = allIngredients.filter(
+        (ingredient) => !selectedIngredients.includes(ingredient)
+      );
+      return excludedIngredients;
+    });
+
+    setExcludedIngredients(excludedIng);
+  }, [cart]);
 
-export const confirmationScreen = () => {
   return (
-    <View>
-      <Text>HELLO FROM CONFIRMATION PAGE</Text>
+    <View style={confirmationPageStyles.mainWrapper}>
+      <ScrollView
+        style={confirmationPageStyles.scrollViewStyle}
+        nestedScrollEnabled={true}
+      >
+        <View>
+          {cart.map((item, index) => (
+            <View style={confirmationPageStyles.mainContainer} key={uuid.v4()}>
+              <View style={confirmationPageStyles.imageWithCountStyles}>
+                <Image
+                  source={{
+                    uri: item.dish.imageURL,
+                  }}
+                  style={confirmationPageStyles.imageStyle}
+                />
+                <View style={confirmationPageStyles.quantityContainer}>
+                  <Text style={confirmationPageStyles.quantityItem}>
+                    X{item.quantity}
+                  </Text>
+                </View>
+              </View>
+              <View style={confirmationPageStyles.textItemsContainer}>
+                <View>
+                  <Text style={confirmationPageStyles.itemTitle}>
+                    {item.dish.name}
+                  </Text>
+                </View>
+                {excludedIngredients[index]?.length > 0 ? (
+                  <View style={confirmationPageStyles.excludedIngWrapper}>
+                    <Text style={confirmationPageStyles.withoutIngText}>
+                      Без добавления:
+                    </Text>
+                    <Text style={confirmationPageStyles.ingTextWrapper}>
+                      {excludedIngredients[index]?.join(", ")}
+                    </Text>
+                  </View>
+                ) : null}
+                <View style={confirmationPageStyles.priceAndIconsContainer}>
+                  <Text style={confirmationPageStyles.priceItem}>
+                    {item.dish.price} BYN
+                  </Text>
+                  <Text style={confirmationPageStyles.priceItem2}>
+                    за 1 позицию
+                  </Text>
+                </View>
+              </View>
+            </View>
+          ))}
+          <View style={confirmationPageStyles.lineView}></View>
+          <View style={confirmationPageStyles.itemsWrapperContainer}>
+            <View style={confirmationPageStyles.itemWrapperStyle}>
+              <Text style={confirmationPageStyles.firstPartTextStyle}>
+                Тип заказа:
+              </Text>
+              <Text style={confirmationPageStyles.secondPartTextStyle}>
+                {orderData.orderType}
+              </Text>
+            </View>
```

Comment:
move to the common component and reuse after

Reply:
fixed